.appveyor.yml

```diff
@@ -4,8 +4,6 @@ os: Visual Studio 2015
 
 environment:
   matrix:
-    #- PYTHON: "C:\\Python35"
-    #- PYTHON: "C:\\Python35-x64"
```

Comment:
Can we remove instead of uncomment? We could also include Python 3.8 here

Reply:
How, but that's exactly what you did: it was commented out before and you removed it. Thanks.